### DISCUSSION TITLE
[Feature] Merge recognized faces

### DISCUSSION THREAD

**Discussion by omarcinp** [immich-app/immich#6675] [Feature Request]
## why
this Is a feature that could allow a better experience while using face recognition features as sometimes same person due too look changes may be recognized as 2 or 3 different persons. So the idea is to have a way to Marge different Face IDs together. It could be automatically by name (Same name is same person) or it could be a label when appearing when writing a name that suggests to merge faces if it matches an already existing face name

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
You can already do this when viewing a person via the menu in the top right.
